### PR TITLE
Updated checklistService

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

File: Service/CreateMarriageService.php [hunk 1]
```diff
@@ -289,7 +289,7 @@ private function createMarriage(array $huwelijk): ?array
             $huwelijkObject->hydrate($requesterAssent);
 
             // @TODO update checklist with moment
-//            $huwelijkObject = $this->updateChecklistService->checkHuwelijk($huwelijkObject);
```
Comment (rjzondervan) [inline Service/CreateMarriageService.php:292]: Don't forget to remove your TODOs

File: Service/CreateMarriageService.php [hunk 2]
```diff
@@ -289,7 +289,7 @@ private function createMarriage(array $huwelijk): ?array
...
+            $huwelijkObject = $this->updateChecklistService->checkHuwelijk($huwelijkObject);
```
Comment (WilcoLouwerse) [inline Service/CreateMarriageService.php:291]: Can we remove this @TODO now?

File: Service/InvitePartnerService.php [hunk 3]
```diff
@@ -152,7 +152,7 @@ private function invitePartner(array $huwelijk, ?string $id): ?array
             $huwelijkObject->hydrate($requesterAssent);
 
             // @TODO update checklist with partners
-//            $huwelijkObject = $this->updateChecklistService->checkHuwelijk($huwelijkObject);
+            $huwelijkObject = $this->updateChecklistService->checkHuwelijk($huwelijkObject);
```
Comment (WilcoLouwerse) [inline Service/InvitePartnerService.php:154]: Can we remove this @TODO now?

File: Service/InviteWitnessService.php [hunk 4]
```diff
@@ -148,7 +148,7 @@ private function inviteWitness(array $huwelijk, ?string $id): ?array
             $huwelijkObject->setValue('getuigen', $witnessAssents);
 
             // @TODO update checklist with getuigen
-//            $huwelijkObject = $this->updateChecklistService->checkHuwelijk($huwelijkObject);
+            $huwelijkObject = $this->updateChecklistService->checkHuwelijk($huwelijkObject);
```
Comment (WilcoLouwerse) [inline Service/InviteWitnessService.php:150]: Can we remove this @TODO now?

**Review by rjzondervan — APPROVED** [review]

File: Service/UpdateChecklistService.php [hunk 5]
```diff
@@ -2,19 +2,39 @@
...
 {
+    /**
+     * @var EntityManagerInterface
+     */
+    private EntityManagerInterface $entityManager;
+
+    /**
+     * @var SymfonyStyle
+     */
     private SymfonyStyle $io;
```
Comment (WilcoLouwerse) [inline Service/UpdateChecklistService.php:23]: Codacy: "$io is a very short name, variable names should be at least 3 characters long"

Maybe use $symfonyStyle ?

**Review by WilcoLouwerse — APPROVED** [review]
Docblocks can use some extra comments

**Pull request merged**